src/components/biggive-container-card/biggive-container-card.scss

```diff
@@ -0,0 +1,32 @@
+@include backgrounds();
+@include fills();
+@include spacers();
+@include text-colours();
```

Comment:
there's a bit of duplication here and in the next file from the biggive-basic-card css file, but it might be worth waiting till we've confirmed the design and then think about if/how we can DRY it.